torch/_torch_docs.py

```diff
@@ -8928,6 +8928,50 @@ def merge_dicts(*dicts):
 """.format(**factory_common_args))
 
 
+add_docstr(torch.kaiser_window,
+           """
+kaiser_window(window_length, beta, periodic=True, dtype=None, \
+layout=torch.strided, device=None, requires_grad=False) -> Tensor
+""" + r"""
+Kaiser window function.
+
+.. math::
+    w[n] = I_0 \left( \beta \sqrt{1 - \frac{4n^2}{(M-1)^2}} \right) / I_0( \beta )
+
+with
+
+.. math::
+    -\frac{M-1}{2} \leq n \leq \frac{M-1}{2}
```

Comment:
I don't understand this bit in the original NumPy documentation. What is this trying to say?